### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by BobVerheij**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,76 @@
 # team-awesome-bob
 
+## Team Awesome
 
+This project is the front end to the future Team Awesome application which will allow enterprises to track time registration of their employees.
```
Comment (RubenWerdmuller) [inline README.md:5]: Do you track time registration?

File: README.md [hunk 2]
```diff
@@ -1,37 +1,76 @@
...
-                                                                                
+You can add new Time Entries and Team Members with some limitations. You can filter these entries and members by some primary variables of these objects.
+
+## Installation
+
+To get a demo of this front end web app, check out this repo and use
+
+```bash
+npm run dev
+```
```
Comment (RubenWerdmuller) [inline README.md:15]: You'll want to install the node modules first

File: README.md [hunk 3]
```diff
@@ -1,37 +1,76 @@
...
+
+### Stack
+
+HTML, CSS, JS ...
+Made with React, styled-components, Zustand, StoryBook, Jest & Cypress
```
Comment (RubenWerdmuller) [inline README.md:20]: No need to mention HTML, CSS and JS. React can be replaced with Next.js, as it's a more pronounced part of the app. Finally, if you mention your used stack, it's nice to add url's so people can check it out.

File: README.md [hunk 4]
```diff
@@ -1,37 +1,76 @@
...
+## Usage
+
+```bash
+npm run storybook		// will start up storybook, to check out the usage of some components.
+```
+
+```bash
+npm run test		// will run Jest with a Watcher.
+npm run cypress		// will fire up Cypress with some End to End test options.
+```
```
Comment (RubenWerdmuller) [inline README.md:31]: I would separate these. While they both perform tests, they are mutually exclusive

File: README.md [hunk 5]
```diff
@@ -1,37 +1,76 @@
...
+### Stack
+
+HTML, CSS, JS ...
+Made with React, styled-components, Zustand, StoryBook, Jest & Cypress
+
+## Usage
+
+```bash
+npm run storybook		// will start up storybook, to check out the usage of some components.
+```
```
Comment (RubenWerdmuller) [inline README.md:26]: I would personally describe the function outside of the code

File: README.md [hunk 6]
```diff
@@ -1,37 +1,76 @@
...
+
+```bash
+npm run test		// will run Jest with a Watcher.
+npm run cypress		// will fire up Cypress with some End to End test options.
+```
+
+### Contributions
+
+Many thanks to [Humanoids](https://github.com/humanoidsbv) for giving me the chance to follow this crash course.
```
Comment (RubenWerdmuller) [inline README.md:35]: This is the first time the crash course is mentioned and might be a irrelevant

File: README.md [hunk 7]
```diff
@@ -1,37 +1,76 @@
...
+
+Big Boss - [Ruben Werdmuller](https://github.com/RubenWerdmuller)
```
Comment (RubenWerdmuller) [inline README.md:37]: <3

File: README.md [hunk 8]
```diff
@@ -1,37 +1,76 @@
...
+                           **,,*(#((/        ,((((((((#(/**,,,#%# KIPRUND
+                           ,*. ./((((.         *###((####(*,,,/%%* KIPRUND
+                            *,.,.////(          %% ###(### /***#%%, KIPRUND
+                            ,..,* //*//              ////  (*,, #%, KIPRUND
+                           *,.., /////(            (/////  *,..*(. KIPRUND
+                         (#(/** %%##((            %%#((*  (/**. KIPRUND
+                                                          .,* KIPRUND
+
+Interested in the Front End CrashCourse, check out:
+[Humanoids](https://humanoids.nl/training/)
```
Comment (RubenWerdmuller) [inline README.md:76]: Nice!

**Review by RubenWerdmuller — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BobVerheij) [reply README.md:5]: Yes

**Pull request merged**